src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -12764,10 +12764,7 @@ Unit* Creature::SelectVictim()
         return NULL;
 
     if (target && _IsTargetAcceptable(target) && CanCreatureAttack(target))
-    {
-        SetInFront(target);
         return target;
```

Comment:
> But, if a player is tanking an NPC and NPC cast spell on another player, without this change it would have had his serverside orientation looking at the tank.

and exactly this is the issue of #10803